src/field_theory/primitive_element.lean

```diff
@@ -0,0 +1,206 @@
+/-
+Copyright (c) 2020 Thomas Browning and Patrick Lutz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Browning and Patrick Lutz
+-/
+
+import field_theory.adjoin
+import field_theory.separable
+
+/-!
+# Primitive Element Theorem
+
+In this file we prove the primitive element theorem.
+
+## Main results
+
+- `exists_adjoin_simple_eq_top`: a finite separable extension has a primitive element, i.e.
+  there is an `α ∈ E` such that `F⟮α⟯ = (⊤ : subalgebra F E)`.
+-/
```

Comment:
```suggestion

## Implementation notes

In declaration names, `primitive_element` abbreviates `adjoin_simple_eq_top`:
it stands for the statement `F⟮α⟯ = (⊤ : subalgebra F E)`. We did not add an extra
declaration `is_primitive_element F α := F⟮α⟯ = (⊤ : subalgebra F E)` because this
requires more unfolding without much obvious benefit.

## Tags

primitive element, separable field extension, separable extension, intermediate field, adjoin, exists_adjoin_simple_eq_top

-/
```